handling-null-values-in-avro-and-protobuf/kafka/.gitignore

```diff
@@ -0,0 +1,47 @@
+# Default ignored files
```

Comment:
since everything in here is petty generic (gradle, intellij, etc), you can add to the root level `.gitignore` as needed and delete this file